ocs_ci/deployment/hosted_cluster.py

```diff
@@ -384,36 +428,16 @@ def __init__(self, name):
                 f"Cluster path for desired cluster with name '{self.name}' was not found in ENV_DATA.clusters"
             )
 
-    def deploy_ocp(
-        self,
-        deploy_cnv=True,
-        deploy_acm_hub=True,
-        deploy_metallb=True,
-        download_hcp_binary=True,
-        deploy_mce=True,
-    ):
+    def deploy_ocp(self, **kwargs) -> str:
         """
         Deploy hosted OCP cluster on provisioned Provider platform
 
         Args:
-            deploy_cnv: (bool) Deploy CNV
```

Comment:
It would be good to keep some arguments documented and have **kwargs after they are listed.

Reply:
we don't use kwargs in any call, this was removed entirely, because it was bad design. Now before calling `deploy_ocp` we need to call `deploy_dependencies`, that reduces shared dependencies and makes code more flexible

Reply:
kwargs were used because deploy_ocp is overloaded method and pylint urges on missing arguments, and kinda for future use

Reply:
Ok, sounds reasonable.